lang/en/texts/faq.html

```diff
@@ -29,19 +29,22 @@ <h2 id="data_verification">Is the information and data on products verified?</h2
 <p>To detect potential errors more easily, we will progressively add automated checks. e.g. if the nutrition facts of a product are very different from
 products of the same category, it may be an error. If the nutriments add up to more than 100%, it's definitely an error.</p>
 
+<p>Finally, many producers send us photos and data about their products using our Platform for Producers. If you are a producer, please <a href="mailto:producers@openfoodfacts.org">contact us</a></p>
+
 
 <h2 id="pictures_and_data_from_other_sites">Can I add product pictures or data from the manufacturer's site, shopping sites or other sites?</h2>
 
-<p>Almost all other sites forbid reproduction and reuse of their data and images, and that's actually the reason why we are creating Open Food Facts: to make all this data available to all and for all uses.</p>
+<p>Probably No. Almost all other sites forbid reproduction and reuse of their data and images, and that's actually the reason why we are creating Open Food Facts: to make all this data available to all and for all uses.</p>
 
 <p>To avoid any legal problem, we therefore ask contributors to only add pictures that they took themselves, and only data that is coming from the product packaging and label.</p>
 
-
 <h2 id="adding_my_products">I am a food product manufacturer, can I add my own products?</h2>
 
 <p>Yes! The only condition is to accept that the data and the pictures be made available under an open licence. (see the <a href="https://en.openfoodfacts.org/terms-of-use#contribution">Terms of contribution</a>)</p>
 
-<p>If you have a large number of products to add (more than 50), we can try to find a way to automatically add the products and their data. Contact-us to discuss how we could proceed: contact@openfoodfacts.org</p>
+<p>We have developped a free platform to let you easily import photos and data about your products, regardless of your size. 
+It will let you import complete and up-to-date data and visuals into Open Food Facts and be visible in more than 100 other nutritional apps. Additionaly, the platform provides suggestions to improve the Nutri-Score of your products. 

```

Comment:
```suggestion
It will let you import complete and up-to-date data and visuals into Open Food Facts that will be visible in more than 100 other nutritional apps. Additionaly, the platform provides suggestions to improve the Nutri-Score of your products. 

```